src/app/app.component.ts

```diff
@@ -12,4 +13,8 @@ export class AppComponent {
 
   width = 400;
   height = 400;
+
+  onDropped(event: Activity) {
```

Comment:
not sure if this should be `event` or `activity`

Reply:
done